script.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# Fail fast on errors, unset variables, and failures in piped commands
+set -Eeuo pipefail
+
+cd "${GITHUB_WORKSPACE}/${INPUT_WORKING_DIRECTORY}" || exit
+
+echo '::group::Preparing ...'
+  unameOS="$(uname -s)"
+  case "${unameOS}" in
+    Linux*)     os=linux;;
+    Darwin*)    os=darwin;;
+    CYGWIN*)    os=windows;;
+    MINGW*)     os=windows;;
+    MSYS_NT*)   os=windows;;
+    *)          echo "Unknown system: ${unameOS}" && exit 1
+  esac
+
+  unameArch="$(uname -m)"
+  case "${unameArch}" in
+    x86*)      arch=amd64;;
+    *)         echo "Unsupported architecture: ${unameArch}. Only AMD64 is supported by tfsec" && exit 1
+    esac
+
+  TEMP_PATH="$(mktemp -d)"
+  echo "Detected ${os} running on ${arch}, will install tools in ${TEMP_PATH}"
+  REVIEWDOG_PATH="${TEMP_PATH}/reviewdog"
+  TFSEC_PATH="${TEMP_PATH}/tfsec"
+echo '::endgroup::'
+
+echo "::group::🐶 Installing reviewdog (${REVIEWDOG_VERSION}) ... https://github.com/reviewdog/reviewdog"
+  curl -sfL https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh | sh -s -- -b "${REVIEWDOG_PATH}" "${REVIEWDOG_VERSION}" 2>&1
+
+  echo "${REVIEWDOG_PATH}" >> "${GITHUB_PATH}"
+  export PATH="${REVIEWDOG_PATH}:${PATH}"
+echo '::endgroup::'
+
+echo '::group:: Installing tfsec (latest) ... https://github.com/tfsec/tfsec'
+  test ! -d "${TFSEC_PATH}" && install -d "${TFSEC_PATH}"
+
+  binary="tfsec"
+  url="https://github.com/tfsec/tfsec/releases/latest/download/tfsec-${os}-${arch}"
+  if [[ "${os}" = "windows" ]]; then
+    url+=".exe"
+    binary+=".exe"
+  fi
+
+  curl --silent --show-error --fail \
+    --location "${url}" \
+    --output "${binary}"
+  install tfsec "${TFSEC_PATH}"
+
+  echo "${TFSEC_PATH}" >> "${GITHUB_PATH}"
```

Comment:
ditto

Reply:
Done in https://github.com/reviewdog/action-tfsec/pull/15/commits/9f98aa381949f1d2db7307563ade9b3384f7d880